Sources/ApplicativeRouter/SyntaxRouter.swift

```diff
@@ -41,7 +41,8 @@ public struct Router<A> {
   }
 
   public func absoluteString(for a: A) -> String? {
-    return url(for: a).map { "/" + $0.absoluteString }
+    return (self.url(for: a)?.absoluteString)
+      .map { $0 == "/" ? "/" : "/" + $0 }
```

Comment:
a bit hacky, but this prevents the double `//` printing bug. think this is related to the `? (base ?? URL(string: "/"))` line in the `request(from: RequestData)` helper method.

this may be the right way to do it, or we can fix later...